src/components/AdminPanel.js

```
@@ -0,0 +1,664 @@
+                  <TableCell>Question</TableCell>
+                  <TableCell>Date</TableCell>
+                </TableRow>
+              </TableHead>
+              <TableBody>
+                {questions.map((q) => (
+                  <TableRow key={q.id}>
+                    <TableCell>{q.fullName || q.authorName || '-'}</TableCell>
+                    <TableCell>{q.email || '-'}</TableCell>
+                    <TableCell>{q.subject || q.title || '-'}</TableCell>
+                    <TableCell>{q.question || q.content || '-'}</TableCell>
+                    <TableCell>{q.createdAt && q.createdAt.toDate ? q.createdAt.toDate().toLocaleString() : '-'}</TableCell>
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)